frontend/components/tooltip/BaseTooltip.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+    <div class="flex flex-col responsive-px-2 responsive-py-1 rounded shadow-md shadow-zinc-700 bg-light-distinct dark:bg-dark-distinct w-max text-light-content dark:text-dark-content tooltip">
+        <slot name="header">
```

Comment:
These extra slots are also not needed, but no stress. Easy to take them out :) :)